operators/src/processing/raster_vector_join/points.rs

```diff
@@ -246,14 +247,15 @@ impl<P: Pixel> PointRasterJoiner<P> {
 }
 
 #[async_trait]
-impl VectorQueryProcessor for RasterPointJoinProcessor {
-    type VectorType = MultiPointCollection;
+impl QueryProcessor for RasterPointJoinProcessor {
```

Comment:
why is it better to use `QueryProcessor` here and define the Output and SpatialBounds manually when you can just use the VectorQueryProcessor which defines at least the latter one for you? The query-method is still available even if you implement VectorQueryProcessor here

Reply:
the same holds for the other processors you changed

Reply:
It isn't better, it is just still possible. You can use both when they fit best.

Reply:
then why change it if it isn't better? It's easier to understand the processor if you use the more precise type. Also please merge the master into this pr,

Reply:
I tried it, you can have problems using query on non-boxed types that implement just VectorQueryProcessor or RasterQueryProcessor since we can only implement
```
impl QueryProcessor for T
where T: RasterQueryProcessor
```
once. If you implement QueryProcessor, it automatically implements VectorQueryProcessor or RasterQueryProcessor that works all the time. So it is just one additional line and it works in more cases.